test/common/common/assert2_test.cc

```diff
@@ -1,10 +1,5 @@
-#include "source/common/common/assert.h"
```

Comment:
cc @ravenblackx i guess this is actually required but is included transitively from utility